service/redirectHandler.go

```diff
@@ -46,6 +47,8 @@ func (rh *RedirectHandler) ServeHTTP(response http.ResponseWriter, request *http
 		return
 	}
 
+	instance += strings.TrimRight(request.URL.Path, "/") //keep original path with trailing '/' chars removed
```

Comment:
use `strings.TrimRight(request.RequestURI, "/")` instead.  Since `RequestURI` is what's sent in the HTTP request initial line, it will include any query string and `#` markers.

Reply:
@johnabass Sounds good. Thanks for clarifying that.